modules/serverless-apiserversource.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="serverless-apiserversource_context"]
+= Using the API server source
+
+The API server source can be used to connect an event sink (for example, a Knative service) to the Kubernetes API server. The `ApiServerSource` watches for Kubernetes events and forwards them to the Knative Eventing broker.
+
+.Example ApiServerSource YAML
+
+----
+apiVersion: sources.knative.dev/v1alpha1
+kind: ApiServerSource
+metadata:
+  name: testevents
+  namespace: default
+spec:
+  serviceAccountName: events-sa
+  mode: Resource
+  resources:
+    - apiVersion: v1
+      kind: Event
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: Broker
```

Comment:
why do we use the broker here, instead of a vanilla Knative Serving Service?

Reply:
If we want to show the broker here, change the `apiVersion` to `eventing.knative.dev/v1beta1`

Reply:
A **very** good point; thank you. This actually makes me not stuck on the broker stuff :slightly_smiling_face: